src/leap/common/http.py

```diff
@@ -76,17 +196,27 @@ def __init__(self, cert_file=None, pool=_pool):
         :param pool: An optional dedicated connection pool to override the
                      default shared one.
         :type pool: HTTPConnectionPool
+        :param timeout: The amount of time that this Agent will wait for the
```

Comment:
I would call this connectionTimeout, to differentiate from above, or we want it to be the same?